Build/Props/CodeJam.Targeting.props

```diff
@@ -21,10 +21,10 @@
 	<!-- VS does select the first moniker in list as default target
 	     therefore we place our default platform (netcoreapp3.0) at start of the list -->
 	<PropertyGroup>
-		<CopyMeTargetFrameworks>netcoreapp3.0;net5.0;netcoreapp2.0;netcoreapp1.0;netstandard2.1;netstandard2.0;netstandard1.5;netstandard1.3;net472;net461;net45;net40;net35;net20</CopyMeTargetFrameworks>
-		<CopyMeTestTargetFrameworks>netcoreapp3.0;net5.0;netcoreapp2.1;netcoreapp1.1;net472;net461;net45;net40;net35;net20</CopyMeTestTargetFrameworks>
-		<CopyMeMinimalTargetFrameworks>netcoreapp3.0;netstandard2.0;net461</CopyMeMinimalTargetFrameworks>
-		<CopyMeMinimalTestTargetFrameworks>netcoreapp3.0;netcoreapp2.1;net461</CopyMeMinimalTestTargetFrameworks>
+		<CopyMeTargetFrameworks>netcoreapp3.1;net5.0;netstandard2.1;netstandard2.0;net472;net461;net45;net40;net35</CopyMeTargetFrameworks>
```

Comment:
There is no code in the library that requires .NET Core 3.1, so there is no real point to require 3.1 instead of 3.0.

Reply:
Targeting Core 1/2/3.0 under msbuild 16.8+ leads to build warning about deprecation.

Reply:
Let's suppress the warning for a while.
I am sure that there are still projects with 3.0.
We can make a scheduled milestone, let's say in half year to remove .NET Core 3.0 support completely.

If you can find out .NET Core version usage in the world it can be a good data for deciding what to drop.

Reply:
ntstd2.1 target still supported